config/environments/production.rb

```diff
@@ -39,6 +42,21 @@
   # Use default logging formatter so that PID and timestamp are not suppressed.
   config.log_formatter = Logger::Formatter.new
 
+  # Mount Action Cable outside main process or domain.
+  # config.action_cable.mount_path = nil
+  # config.action_cable.url = "wss://example.com/cable"
+  # config.action_cable.allowed_request_origins = [ "http://example.com", /http:\/\/example.*/ ]
+
+  # Assume all access to the app is happening through a SSL-terminating reverse proxy.
+  # Can be used together with config.force_ssl for Strict-Transport-Security and secure cookies.
+  # config.assume_ssl = true
+
+  # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
+  # config.force_ssl = true
```

Comment:
From the [documentation](https://guides.rubyonrails.org/v7.2/configuring.html#config-force-ssl):

> Forces all requests to be served over HTTPS, and sets "https://" as the default protocol when generating URLs.

This kind of redirect already happens in apache via this config:

```conf
<IfModule mod_rewrite.c>
    RewriteEngine On
    RewriteCond %{HTTPS} off
    RewriteRule (.*) https://%{HTTP_HOST}%{REQUEST_URI} [R=301,L]
</IfModule>
```

so why not also use it directly in Rails such that all generated URLs automatically have `https://` as default protocol.

Reply:
Config option activated via commit 19d726beec.